images/node/.dockerignore

```diff
@@ -0,0 +1,21 @@
+# Ignore node_modules directory (dependencies will be installed in the Docker image)
+node_modules/
```

Comment:
Lets add 

.DS_Store
.AppleDouble
.LSOverride

Reply:
Mac puts funky things in the folders to remember layout of files in finder, I think